src/views/Stock/ListProductStock.js

```diff
@@ -163,7 +163,8 @@ const StockRow = ({
       onSetQuantity(product, debouncedQuantity);
     }
     setAdd(0);
-  }, [debouncedQuantity, onSetQuantity, product]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [debouncedQuantity]);
```

Comment:
add [debouncedQuantity] here will trigger infinite loop request in your chrome -> network